tests/test_meta_tag_hidden.py

```diff
@@ -0,0 +1,118 @@
+from tests.pyxform_test_case import PyxformTestCase
+
+"""
+Test hidden meta tags functionality in pyxform.
+Tests that meta elements get the tag="hidden" attribute for CHT SMS functionality.
+"""
+
+
+class MetaTagHiddenTest(PyxformTestCase):
+    """Test cases for hidden meta tags."""
+
+    def test_meta_tag_hidden_by_default(self):
+        """Test that meta elements get tag='hidden' attribute by default."""
+        self.assertPyxformXform(
+            name="test_meta",
+            md="""
+            | survey |        |         |       |
+            |        | type   | name    | label |
+            |        | text   | q1      | Q1    |
+            """,
+            xml__contains=[
+                '<meta tag="hidden">',
+                "<instanceID/>",
+                "</meta>",
+            ],
+        )
+
+    def test_meta_tag_with_audit(self):
+        """Test that meta tag='hidden' is present when audit field is used."""
+        self.assertPyxformXform(
+            name="test_audit",
+            md="""
+            | survey |        |         |       |
+            |        | type   | name    | label |
+            |        | audit  | audit   |       |
+            """,
+            xml__contains=[
+                '<meta tag="hidden">',
+                "<audit/>",
+                "<instanceID/>",
+                "</meta>",
+            ],
+        )
+
+    def test_meta_tag_with_sms_attributes(self):
+        """Test that meta tag='hidden' is present with CHT SMS prefix and delimiter."""
+        self.assertPyxformXform(
+            name="test_sms",
+            md="""
+            | survey |        |         |       |
+            |        | type   | name    | label |
+            |        | text   | q1      | Q1    |
+            """,
+            xml__contains=[
+                '<meta tag="hidden">',
+                'prefix="J1!data!"',  # Default survey name is "data"
+                'delimiter="#"',
+                "<instanceID/>",
+                "</meta>",
+            ],
+        )
```

Comment:
Can probably remove these tests since they are not really adding anything different on top of what is already covered in the first test.